gcm4/src/test/java/plugins/personproperties/datamanagers/AT_PersonPropertyDataManager.java

```diff
@@ -1334,6 +1333,158 @@ public boolean equals(Object obj) {
 
 	}
 
+	@Test
+	@UnitTestMethod(target = PersonPropertiesDataManager.class, name = "getEventFilterForPersonPropertyUpdateEvent", args = {
+			PersonPropertyId.class, Object.class, boolean.class })
+	public void testGetEventFilterForPersonPropertyUpdateEvent_propertyId_object() {
+
+		RandomGenerator randomGenerator = RandomGeneratorProvider.getRandomGenerator(8796864982253772625L);
+
+		Set<MultiKey> expectedObservations = new LinkedHashSet<>();
+		Set<MultiKey> actualObservations = new LinkedHashSet<>();
+		TestPluginData.Builder pluginBuilder = TestPluginData.builder();
+
+		// get testPropertyIds to use
+		ArrayList<TestPersonPropertyId> testPersonPropertyIds = new ArrayList<>();
+
+		for (TestPersonPropertyId testPersonPropertyId : TestPersonPropertyId.values()) {
+			if (testPersonPropertyId.getPropertyDefinition().propertyValuesAreMutable()) {
+				testPersonPropertyIds.add(testPersonPropertyId);
+			}
+		}
+
+		// set and subscribe to test actor plans for each testPropertyId
+		for (TestPersonPropertyId testPersonPropertyId : testPersonPropertyIds) {
+
+			// generate 50 random values
+			ArrayList<Object> allValues = new ArrayList<>();
+			for (int i = 0; i < 50; i++) {
+				Object value = testPersonPropertyId.getRandomPropertyValue(randomGenerator);
+				allValues.add(value);
```

Comment:
This is broken, as one of the testPersonPropertyIds is of type Boolean, meaning there are only 2 possible values in that instance, so you would have a list full of trues and falses which throws an error in the even subscription.